src/DIRAC/Core/Utilities/MySQL.py

```diff
@@ -942,10 +942,14 @@ def _to_value(self, param):
     """
     return str(param[0])
 
-  def _to_string(self, param):
+  def _to_string(self, value):
     """
+      Convert value to string
     """
-    return param[0].tostring()
+    try:
+      return value.decode()
+    except (UnicodeDecodeError, AttributeError):
+      return str(value)
```

Comment:
If there is a `UnicodeDecodeError` then `return str(value)` is definitely the wrong thing to do as we'll get junk that starts to spread around. I'm also no sure when `AttributeError` would trigger (`value` is already a string maybe)?
```suggestion
    return value.decode()
```

Reply:
Looking further down this PR: I think `_to_string(xxx)` can be deleted in favour of using `xxx.decode()`

Reply:
looking at the scope of this method, if only to convert a variable that is a string, then agree